### PR TITLE
Fix broken build for python-zipp

### PR DESCRIPTION
**Pull request by dennisameling**
Currently, building `python-zipp` fails with:

```shell
Dennis@DESKTOP-0RP8NC0 MSYS /usr/src/MINGW-packages/mingw-w64-python-zipp
$ MINGW_ARCH=clang64 makepkg-mingw -sLf --skippgpcheck
==>  MINGW_ARCH: clang64
  -> Building clang64...
==> Making package: mingw-w64-python-zipp 3.1.0-2 (Wed Dec 15 22:17:29 2021)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Installing missing dependencies...
error: target not found: mingw-w64-clang-x86_64-python-unittest2
==> ERROR: 'pacman' failed to install missing dependencies.
==> Missing dependencies:
  -> mingw-w64-clang-x86_64-python-unittest2
==> ERROR: Could not resolve all dependencies.
```

The test dependencies in `setup.cfg` [have changed](https://github.com/jaraco/zipp/compare/v2.0.0...v2.2.1#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52) between versions 2.0.0 and 2.2.1 (MINGW-packages is at 3.1.0 currently)

```
pathlib2
contextlib2
unittest2
```

... became

```
jaraco.itertools
func-timeout
```

This PR adds the new dependencies. Please advise whether I added the new packages the right way - the build works and `python-zipp` tests are passing, but you might have some feedback on how the `PKGBUILD` files are set up (I copied them from another existing Python package)

### CODE REVIEW

**Comment by jeremyd2019** [comment]
Thanks

I think that packages generally have `.` changed to `-` in their names here, but maybe somebody who does more python packaging has an opinion.

File: mingw-w64-python-jaraco.itertools/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+pkgver=6.0.3
+pkgrel=1
+pkgdesc='Jaraco Itertools'
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
+url="https://github.com/jaraco/jaraco.itertools"
+license=('MIT')
+depends=(
+  "${MINGW_PACKAGE_PREFIX}-python-more-itertools"
```
Comment (jeremyd2019) [inline mingw-w64-python-jaraco.itertools/PKGBUILD:15]: It looks like python-more-itertools needs to be enabled for clangarm64 too

**Comment by jeremyd2019** [comment]
I used AWS to provide an ARM64 runner for the CI, and turned it loose on all of your PRs merged together into one branch.  https://github.com/msys2-arm/MINGW-packages/runs/4542133096?check_suite_focus=true.  That's how I found that more-itertools seems to still be missing.

**Comment by dennisameling** [comment]
Thanks for checking! In contrast to my other PRs, I didn't test this one on arm64 yet as its main purpose is to fix the broken build on x64 to start with. Just enabled `more-itertools` for `clangarm64` in https://github.com/msys2/MINGW-packages/pull/10341/commits/41ecd4bfea62a15aa534ca11e0a6aae103abae0f and that indeed allowed me to build `python-zipp` for `clangarm64` as well 🎉 

```
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Stripping unneeded symbols from binaries and libraries...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "mingw-w64-clang-aarch64-python-zipp"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Finished making: mingw-w64-python-zipp 3.1.0-2 (Thu Dec 16 09:46:43 2021)
```

**Comment by jeremyd2019** [comment]
> I think that packages generally have `.` changed to `-` in their names here, but maybe somebody who does more python packaging has an opinion.


@lazka or someone want to weigh in on the name?  I feel it should be python-jaraco-itertools

Also:
```
	message: mingw-w64-python-func-timeout: missing description suffix ‘(mingw-w64)’
	message: mingw-w64-python-inflect: missing description suffix ‘(mingw-w64)’
	message: mingw-w64-python-jaraco.itertools: missing description suffix ‘(mingw-w64)’
```

**Comment by raedrizqie** [comment]
all current jaraco submodules uses jaraco.* on our repo.. same as pypi.. but i prefer using - or _

**Comment by Biswa96** [comment]
IMO, follow archlinux naming scheme https://archlinux.org/packages/?sort=&q=jaraco&maintainer=&flagged=

**Comment by jeremyd2019** [comment]
ok, didn't see the precedent.

**Comment by lazka** [comment]
yeah, if unclear go with Arch. Makes matching up packages with Arch easier.

File: mingw-w64-python-func-timeout/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Maintainer: ???
+
+_pyname=func_timeout
+_realname=func-timeout
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+pkgver=4.3.5
+pkgrel=1
+pkgdesc='Python module which allows you to specify timeouts when calling any existing function, and support for stoppable threads'
```
Comment (jeremyd2019) [inline mingw-w64-python-func-timeout/PKGBUILD:9]:
```suggestion
pkgdesc='Python module which allows you to specify timeouts when calling any existing function, and support for stoppable threads (mingw-w64)'
```

File: mingw-w64-python-inflect/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Maintainer: ???
+
+_pyname=inflect
+_realname=inflect
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+pkgver=5.3.0
+pkgrel=1
+pkgdesc='Inflect'
```
Comment (jeremyd2019) [inline mingw-w64-python-inflect/PKGBUILD:9]:
```suggestion
pkgdesc='Python module to correctly generate plurals, ordinals, indefinite articles; convert numbers to words (mingw-w64)'
```

File: mingw-w64-python-jaraco.itertools/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# Maintainer: ???
+
+_pyname=jaraco.itertools
+_realname=jaraco.itertools
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+pkgver=6.0.3
+pkgrel=1
+pkgdesc='Jaraco Itertools'
```
Comment (jeremyd2019) [inline mingw-w64-python-jaraco.itertools/PKGBUILD:9]:
```suggestion
pkgdesc='Jaraco Itertools (mingw-w64)'
```

Or something more descriptive (I failed to find one)

**Pull request merged**